extra/end-to-end-test.rb

```diff
@@ -0,0 +1,66 @@
+require 'rest-client'
+require 'json'
+require 'pp'
+
+# subnets and domains must exist on Infoblox
+URL = "http://localhost:8448"
+DNS_DOMAIN = "ibx.lan"
+DNS_IPV4 = "192.168.202.13"
```

Comment:
I'd also not use any real IPs. rfc5735 suggests using TEST-NET-1 (`192.0.2.0/24`), TEST-NET-2 (`198.51.100.0/24`) or TEST-NET-3 (`203.0.113.0/24`) for this. Even if it's unlikely that something is routing the NAT network, test nets are intended for this.

Reply:
Changed, these IPs are real IP from my Infoblox setup, but I've introduced ENV variables to override this in this test script.

Reply:
Also changed the domain to "example.com".